zboss/doc/zboss_configuration.rst

```diff
@@ -80,7 +34,7 @@ Both of these features are experimental.
 ZBOSS library certification
 ***************************
 
-Zigbee ZBOSS libraries in |NCS| in the production state are always tested in-house by Nordic Semiconductor to conform to Zigbee Pro R22 test specification.
+Zigbee ZBOSS libraries in |NCS| are always tested in-house by Nordic Semiconductor to conform to Zigbee Pro R22 test specification.
 However, these libraries are only periodically verified by external test service provider and granted Zigbee Compliant Platform certification by Zigbee Alliance.
 
 .. |list_intro| replace:: For a complete list of the ZBOSS configuration options, see the following files:
```

Comment:
Please change |list_outro| to the following:
``These libraries are used in the Zigbee protocol configuration in the |NCS| when defining the Zigbee device role, as described in :ref:`nrf:zigbee_ug_configuration` in the |NCS| documentation.``

Reply:
Proposed change applied.